3 reports.sql

```diff
@@ -0,0 +1,30 @@
+
+
+--1) Show a list of all soldiers with all details sorted by IQ level from high to low.
+select * 
+from soldiers s 
+order by IQLevel desc 
+
+--2) I need a list of all soldiers sorted by age at enlistment from high to low. Do not include columns that are not relevant to this list.
+select LastName, FirstName, Age= year(DateOfEnlistment)-year(DOB)
```

Comment:
There's a better way to check diff in dates.

Reply:
I fixed this